test/components/enketo-iframe.spec.js

```diff
@@ -109,6 +109,28 @@ describe('EnketoIframe', () => {
     fakeAssign.calledWith(new URL('http://example.com/projects/1')).should.be.true;
   });
 
+  it('redirects on submissionsuccess for new submission', async () => {
+    const fakeAssign = sinon.fake();
+    const wrapper = mountComponent({
+      props: { enketoId, actionType: 'new' },
+      container: {
+        router: mockRouter('/?return_url=http://example.com/projects/1'),
+        location: {
+          origin: window.location.origin,
```

Comment:
No change needed, just FYI, I added a location mock such that `container.location.origin` automatically equals `window.location.origin` unless it's overridden, same for other properties. So you should only need to specify the property you need to mock (`assign`).